Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -176,5 +186,15 @@ public class CommonOptionBag {
       options.DefiniteAssignmentLevel = value ? 1 : 2;
     });
 
+    DafnyOptions.RegisterLegacyBinding(ReportFileOption, (options, value) => {
+      options.AuditorReportFile = value;
```

Comment:
Instead of the field `dafnyOptions.AuditorReportFile`, use `dafnyOptions.Get(ReportFileOption)`, and then you don't need to call `RegisterLegacyBinding` either.

Reply:
Will do!